readme.md

```diff
@@ -1,13 +1,35 @@
 # Projects
 This repository holds the source code of the following projects:
+- [Metrics](#bar_chart-metrics)
 - [API Mock Server](#test_tube-api-mock-server)
 - [Guardian](#shield-guardian)
-- [Metrics](#bar_chart-metrics)
 - [Shell Words](#shell-shell-words)
 - [YAML Map](#world_map-yaml-map)
 
 Let's review each of them in a bit more details:
 
+## :bar_chart: Metrics
+[Metrics](metrics/readme.md) is a set of software components to collect and review software project metrics like performance, build stability, and codebase quality. 
+The Metrics project integrates with the following CI tools to collect software project metrics:
+- [Buildkite](https://buildkite.com/docs/tutorials/getting-started)
+- [Github Actions](https://docs.github.com/en/actions/quickstart)
+- [Jenkins](https://jenkins.io/doc/)
+
+The Metrics Web Application displays the next project metrics on the dashboard:
+
+| Metric | Description |
+| --- | --- |
+| `Build Status`   | The result of the latest build. |
+| `Build Results`   | The build results and duration of the last 30 builds. |
```

Comment:
```suggestion
| `Build Results`   | A bar chart displaying the build results and duration of the last 30 builds. |
```
What do you think? Would it be useful to update all metrics descriptions using the following pattern?